.github/workflows/release.yml

```diff
@@ -346,12 +340,26 @@ jobs:
     name: Start syncs to other container registries
     needs: ["kitchen-sink", "base-manifests", "debian-sdk-manifests", "ubi-sdk"]
     runs-on: ubuntu-latest
+    # This workflow can be triggered by 2 events: workflow_dispatch, i.e., a
+    # manual run, or repository_dispatch, which is triggered by a Pulumi
+    # release.  In the case of a new Pulumi release (i.e. repository_dispatch),
+    # we will always want to sync to the other repos.  However, in the event of
+    # a workflow_dispatch we are probably debugging an issue and we do not
+    # necessarily want to automatically sync to the other repos.  Both repo sync
+    # workflows allow for their own manual trigger (i.e. repository_dispatch),
+    # so it's not too much work to run the sync workflows manually.
+    #
+    # This design choice also allows us avoid having to alter pulumictl to
```

Comment:
I don't understand this sentence about pulumictl - the original design also allowed for that?

Reply:
Resolved this question with Paul in a Zoom - pulumictl does not allow for arbitrary arguments to be passed in the `repository_dispatch`.

Marking resolved.